verification/cbmc/include/utils.h

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#pragma once
```

Comment:
Why do we need this here?

Reply:
I added it because most header files have it (in both ESDK and aws-c-common). But removing it works fine, so done.